django2pydantic/handlers/numbers.py

```
@@ -1,154 +1,119 @@
+from django2pydantic.handlers.base import DjangoFieldHandler, TDjangoField_co
 
-class IntegerFieldHandler(DjangoFieldHandler[models.IntegerField[int, int]]):
-    """Handler for Integer fields."""
 
-    @override
-    @classmethod
-    def field(cls) -> type[models.IntegerField[int, int]]:
-        return models.IntegerField
-
-    # TODO(phuongfi91): Revisit all the potential None values in mix and max params
+class DjangoIntegerFieldHandler(
```

Ruff (0.8.2)

Missing docstring in public class

(D101)